controllers/discovery/initDiscoveryUserData.js

```diff
@@ -41,6 +47,51 @@ const InitDiscoveryUserData = async (req, res) => {
         )`
         : 'ARRAY[]::text[]';
 
+    // OLD QUERY
+    /* const usersWithCommonFollowerQuery = `
+      SELECT 
+          A.user_id,
+          A.country_code,
+          A.username,
+          A.real_name,
+          A.created_at,
+          A.updated_at,
+          A.last_active_at,
+          A.status,
+          A.profile_pic_path,
+          A.profile_pic_asset_id,
+          A.profile_pic_public_id,
+          A.bio,
+          A.is_banned,
+          A.is_anonymous,
+          A.combined_user_score,
+          A.karma_score,
+          ${similarTopicQuery} as community_info,
+          CommonUsers.common, 
+          B.user_id_follower 
+      from users A
+      LEFT JOIN 
+              (SELECT 
+                  common.*,
+                  joint.common,
+              CASE WHEN joint.source = :userId THEN 1 ELSE 0 END as user_match
+              FROM users common
+              JOIN 
+                  vwm_user_common_follower_count joint
+              ON common.user_id = joint.target
+              WHERE joint.source = :userId AND common.is_anonymous = false) CommonUsers
+      ON CommonUsers.user_id = A.user_id
+      LEFT JOIN user_follow_user B
+      ON A.user_id = B.user_id_followed AND B.user_id_follower = :userId
+      WHERE A.user_id != :userId AND A.is_anonymous = false AND A.is_banned = false
+      ORDER BY
+      COALESCE(B.user_id_follower, '') DESC,
+      COALESCE(A.karma_score, 0) DESC,
+      COALESCE(CommonUsers.common, -1) DESC, 
+      COALESCE(CommonUsers.user_match, -1) DESC
+      LIMIT :limit
+      OFFSET :offset`; */
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good practice to remove commented-out code to keep the codebase clean and maintainable.
```diff
- // OLD QUERY
- /* const usersWithCommonFollowerQuery = `
-   SELECT 
-       A.user_id,
-       A.country_code,
-       A.username,
-       A.real_name,
-       A.created_at,
-       A.updated_at,
-       A.last_active_at,
-       A.status,
-       A.profile_pic_path,
-       A.profile_pic_asset_id,
-       A.profile_pic_public_id,
-       A.bio,
-       A.is_banned,
-       A.is_anonymous,
-       A.combined_user_score,
-       A.karma_score,
-       ${similarTopicQuery} as community_info,
-       CommonUsers.common, 
-       B.user_id_follower 
-   from users A
-   LEFT JOIN 
-           (SELECT 
-               common.*,
-               joint.common,
-           CASE WHEN joint.source = :userId THEN 1 ELSE 0 END as user_match
-           FROM users common
-           JOIN 
-               vwm_user_common_follower_count joint
-           ON common.user_id = joint.target
-           WHERE joint.source = :userId AND common.is_anonymous = false) CommonUsers
-   ON CommonUsers.user_id = A.user_id
-   LEFT JOIN user_follow_user B
-   ON A.user_id = B.user_id_followed AND B.user_id_follower = :userId
-   WHERE A.user_id != :userId AND A.is_anonymous = false AND A.is_banned = false
-   ORDER BY
-   COALESCE(B.user_id_follower, '') DESC,
-   COALESCE(A.karma_score, 0) DESC,
-   COALESCE(CommonUsers.common, -1) DESC, 
-   COALESCE(CommonUsers.user_match, -1) DESC
-   LIMIT :limit
-   OFFSET :offset`; */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->